src/main/java/lotto/model/BonusNumber.java

```diff
@@ -0,0 +1,36 @@
+package lotto.model;
+
+import java.util.List;
+import lotto.util.ExceptionHelper;
+
+import static lotto.model.Lotto.DUPLICATE_NUMBER_ERROR_MESSAGE;
+import static lotto.model.constant.Lotto.MAX_NUMBER;
+import static lotto.model.constant.Lotto.MIN_NUMBER;
+
+public class BonusNumber {
+    public final static String INVALID_NUMBER_RANGE_ERROR_MESSAGE = "번호는 1 이상 45 이하로 입력해주세요.";
```

Comment:
해당 메시지는 로또 숫자 범위가 변하면 바뀌므로 MIN_NUMBER와 MAX_NUMBER를 이용해서 만들면 좋을것 같아요!
```suggestion
    public final static String INVALID_NUMBER_RANGE_ERROR_MESSAGE = "번호는"  + MIN_NUMBER + " 이상 " + MAX_NUMBER + " 이하로 입력해주세요.";
```

Reply:
이런 식으로 상수 안에 상수를 대입하는건 생각하지 못했네요.
좋은 정보 감사합니다.